docs/tutorial.md

```diff
@@ -0,0 +1,360 @@
+# Repository Template Tutorial
```

Comment:
Could we convert this tutorial to use serverless technology, please? Let's consider using AWS Lambda or Azure Functions, or both.

Reply:
My original plan was to use [mangum](https://mangum.io/) to just mount the fastapi app as a Lambda event handler, but I'm not sure we should be pointing people at using a v0.17 project.  It's been around for a few years - what do you think?